.github/workflows/README.md

```diff
@@ -0,0 +1,27 @@
+# Workflow Management
+
+Here we have the start of a system to automatically generated workflows (currently only for transforms). 
+In general, the design is to use templates and `make` to generate/update the workflows.
+
+```
+make
+git commit -a -s -m "update workflows"
+git push
+```
+
+should be sufficient.
+
+## Transforms 
+We define a unique test workflow for each transform, based on a common template [test-transform.template](test-transform.template).
+The Makefile is used to (re)generate all workflows a necessary.  By design, workflows for a given transform should run when
+
+* anything of substance effecting operation is modified in the transform's directory tree.
+* anything in the core libraries in this repo (e.g., data-processing/lib) assuming the transform depends on these.
+* Help! the workflow should NOT run when documentation (e.g., !**.md) is changed, however this case does not seem to be working atm.
+
```

Comment:
It might be worth mentioning that the kfp workflow of the transform is not tested in the unique test workflow.

Reply:
Add some details. not sure if its what you're looking for.